app/controllers/katello/api/v2/packages_controller.rb

```diff
@@ -13,6 +14,50 @@ def auto_complete_name
       render :json => rpms.pluck(col)
     end
 
+    api :GET, "/hosts/:host_id/packages/applicable", N_("List :resource_id")
+    api :GET, "/:resource_id", N_("List :resource_id")
+    api :GET, "/content_views/:content_view_id/filters/:filter_id/:resource_id", N_("List :resource_id")
+    api :GET, "/content_view_filters/:content_view_filter_id/:resource_id", N_("List :resource_id")
+    api :GET, "/repositories/:repository_id/:resource_id", N_("List :resource_id")
+    param :organization_id, :number, :desc => N_("organization identifier")
+    param :content_view_version_id, :identifier, :desc => N_("content view version identifier")
+    param :content_view_filter_id, :identifier, :desc => N_("content view filter identifier")
+    param :repository_id, :number, :desc => N_("repository identifier")
+    param :environment_id, :number, :desc => N_("environment identifier")
+    param :ids, Array, :desc => N_("ids to filter content by")
+    param :host_id, :number, :desc => N_("host id to list applicable packages for")
+    param :packages_restrict_installable, :boolean, :desc => N_("Only show packages that are installable")
+    param :packages_restrict_applicable, :boolean, :desc => N_("Only show packages that are applicable to hosts (defaults to true if host_id is specified)")
+    param_group :search, ::Katello::Api::V2::ApiController
+    def index
+      super
+    end
+
+    def custom_index_relation(collection)
+      if params[:packages_restrict_installable]
```

Comment:
I'm testing this out with my UI code and it looks like when the UI passes in 'false', it's a string (which evals to true). You probably need to cast it like we do for errata:

https://git.io/vi2UH